android/src/main/java/com/cardio/RNCardIOUtilities.java

```diff
@@ -18,8 +19,10 @@ public String getName() {
     return "RCTCardIOUtilities";
   }
 
-  @ReactMethod
-  public void canReadCardWithCamera(Callback callback) {
-    callback.invoke(CardIOActivity.canReadCardWithCamera());
+  @Override
+  public Map<String, Object> getConstants() {
+    final Map<String, Object> constants = new HashMap<>();
+    constants.put("CAN_READ_CARD_WITH_CAMERA", Boolean.valueOf(CardIOActivity.canReadCardWithCamera()));
```

Comment:
The boxing is unnecessary here (`canReadCardWithCamera` already returns a `boolean`), can you remove it?